packages/wp-story-editor/src/api/utils/snakeToCamelCase.js

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { snakeToCamelCase } from '@web-stories-wp/design-system';
+
+/**
+ * Transform a given object keys from snake case to camel case recursively.
+ *
+ * @param {Object} obj Object to be transformed.
+ * @return {any} Transformed object.
+ */
+export function snakeToCamelCaseObjectKeys(obj) {
```

Comment:
Any reason not to move that to design system as well?

Reply:
Yeah, I was tempted to move it to design-system, but I stopped because it's only being used in `wp-*` packages. Ref https://github.com/google/web-stories-wp/pull/9644#discussion_r745695726

Reply:
Ah right!